content/pages/download-va-letters.md

```diff
@@ -0,0 +1,104 @@
+
+#### Proof of Service
+
+<ul>
+<li>
+<strong>Proof of Service Card</strong><br>
+This card shows that you served honorably in the Armed Forces. You can use it as proof of status to receive discounts at certain stores or restaurants.
+</li>
+<li>
+<strong>Service Verification Letter</strong><br>
+This letter shows your branch of service, the date you started active duty, and the date you were discharged from active duty.
+</li>
+<li>
+<strong>Civil Service Preference Letter</strong><br>
+This letter shows that you’re a disabled Veteran and you qualify for preference for civil service jobs.
+</li>
+</ul>
+
+#### Access to benefits
+
+<ul>
+<li>
+<strong>Commissary Letter</strong><br>
+This letter will help get you access to local base facilities, including the commissary and post exchange.
+</li>
+<li>
+<strong>Proof of Creditable Prescription Drug Coverage</strong><br>
+This letter proves that you qualify for Medicare Part D prescription drug coverage.
+</li>
+<li>
+<strong>Minimum Essential Coverage Letter</strong><br>
+This letter shows that you have Minimum Essential Coverage (MEC) as provided by VA. MEC means that your health care plan meets the health insurance requirements under the Affordable Care Act.
+</li>
+</ul>
+
+#### Summary of benefits
+
+<ul>
+<li>
+<strong>Benefit Summary and Service Verification Letter</strong><br>
+This letter shows your service history and some VA benefits information. You can customize this letter and use it for many things.
+</li>
+<li>
+<strong>Benefit Verification Letter</strong><br>
+This letter shows the benefits you’re receiving from VA.
+</li>
+</ul>
+</div>
+
+### How do I download a VA letter?
+
+Before you download your letter, we’ll ask you to review the address we have on file for you.
+
+This address will be listed on your letter. If this address isn’t correct, you can update it. But your letter will still be valid even with the incorrect address.
+
+**Note:** To download a letter, you’ll need the latest version of Adobe Reader. It’s free to download.
+
+[Get Adobe Reader](https://get.adobe.com/reader/).
+
+### What if I want to download a letter or document that isn’t available from this tool?
+
+Right now, you can only download the VA letters listed above.
+
+Use these links to get access to other common VA letters and documents you may be eligible for:
+
+[View and print your Post-9/11 GI Bill statement of benefits.](/education/gi-bill/post-9-11/ch-33-benefit)
+
+[Sign in to eBenefits to request a Certificate of Eligibility for home loan benefits](https://eauth.va.gov/ebenefits/coe).
+
+[Sign in to eBenefits to request a copy of your discharge or separation papers (DD214)](https://eauth.va.gov/ebenefits/DPRIS).
+
+### What if I have trouble downloading a VA letter?
+
+Call the Vets.gov Help Desk at 1-855-574-7286 (TTY: 1-800-877-8339).
+We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET).
+
+</div>
+</div>
+</div>
```

Comment:
It looks like we may want to add some `<br>` at the end of the file here to make the spacing consistent with the `/families-caregivers` page.